src/hooks/useEffectDebugger.js

```diff
@@ -0,0 +1,36 @@
+import { useEffect, useRef } from "react";
```

Comment:
btw i like the idea of storing such utility hooks in separate place, Helpers are pretty big already 👍